src/BrutalTheme.res

```diff
@@ -169,3 +169,10 @@ let brutalRules = (theme: CardThemeType.themeClass) =>
 
 let default = brutal
 let defaultRules = brutalRules
+
+let defaultButtonRules = {
+  ...PaymentType.defaultValueSdkHandleConfirmPaymentProps,
+  backgroundColor: default.colorPrimary,
+  borderColor: default.borderColor,
+  textColor: default.colorBackgroundText,
+}
```

Comment:
please remove these 

we have the Corresponding variables colorBackground, borderColor, colorPrimaryText

Reply:
If you are changing something for the buttonColor 
i think you can keep the name specific to it like ButtonColorBackGournd
ButtonColor

Reply:
This is a total different object so it won't conflict. I added the button keyword ahead. 

@vsrivatsa-juspay  suggested to make separate down object and use these names.

Reply:
and keep those in the same obj, when merchants try to customize anything they can do it at one place

Reply:
Button is part of SDK now .. we have to keep all the customization in one obj

Reply:
It was discussed in starting only with @akash-kamble-juspay and @vsrivatsa-juspay that we will have 2 props for rendering the SDK Button. That's why it was it was made in this way. 

I guess it's better to divide into as those are theme based things customisation and this is the total button related customization.

@akash-kamble-juspay Any thoughts?